FrontEnd/src/components/ProfilePage/ProfilePage.js

```diff
@@ -3,76 +3,33 @@ import Description from './ProfilePageComponents/Description';
 import ProfileContent from './ProfilePageComponents/ProfileContent';
 import { useState } from 'react';
 import BreadCrumbs from '../BreadCrumbs/BreadCrumbs';
+import Loader from '../loader/Loader';
+import { useUser, useProfile } from '../../hooks/';
 
 
-const USER = {
-    'email': 'ex@gmail.com',
-    'companyEmail': '',
-    'password': '12345678',
-    'confirmPassword': '12345678',
-    'surname': 'Василенко',
-    'name': 'Тарас',
-    'companyName': ' Назва Компанії',
-    'companyCheckbox': true,
-    'startupCheckbox': false,
-    'rulesAgreement': true,
-    'position': '',
-    'brend': '',
-    'companyOfficialName': '',
-    'edrpou': '',
-    'activities': [],
-    'categories': [],
-    'regions': '',
-    'bannerImage': '',
-    'logo': '',
-    'slogan': '',
-    'companyInfo': '',
-    'productInfo': '',
-    'serviceInfo': '',
-    'logisticProductService': '',
-    'cooperationFormat': '',
-    'competitiveAdvantage': '',
-    'foundationYear': '',
-    'companySize': '',
-    'topClients': '',
-    'passedAudit': '',
-    'startupName': '',
-    'investmentAmount': '',
-    'cooperationGoals': [],
-    'endResult': '',
-    'competitiveAdvantageIdea': '',
-    'risks': '',
-    'searchPartners': '',
-    'startupIdea': '' ,
-    'phoneNumber': '',
-    'companySite': '',
-    'address': '',
-    'Facebook': '',
-    'Instagram': '',
-    'Tiktok': '',
-    'LinkedIn': '',
-    'Youtube': '',
-};
-
 const ProfilePage = () => {
-    const [mainUser, setMainUser] = useState(USER);
+    const { user } = useUser();
+    const { profile } = useProfile();
     const [formName, setFormName] = useState('');
+
     const currentFormNameHandler = (currentName) => {
         setFormName(currentName);
     };
 
-    const profileUpdateHandler = (myUser) => {
-        console.log('in app');
-        console.log(myUser);
-        setMainUser((prev) => {
-            return {...prev, ...myUser};
-        });
-    };
     return (
         <div className={css['container']}>
-            <BreadCrumbs currentPage="Профіль"/>
-            <Description companyName={mainUser.companyName} brend={mainUser.brend} formName={formName}/>
-            <ProfileContent user={mainUser} onUpdate={profileUpdateHandler} currentFormNameHandler={currentFormNameHandler} formName={formName}/>
+            <BreadCrumbs currentPage="Профіль" />
+            {!profile
+                ? <Loader />
+                :
+                <>
```

Comment:
No need for the fragment

Reply:
fixed)

Reply:
or not

Reply:
I got an error: Parsing error: Adjacent JSX elements must be wrapped in an enclosing tag. Did you want a JSX fragment <>...</>?. That is why i used a fragment in ProfilePage.

Reply:
Oh, in this case, you need a fragment I've seen places where only one element was wrapped in the fragment it's redundant. You need to use it when you have more than one element in the expression